docs/ref-info/ref-tokens.md

```diff
@@ -9,21 +9,20 @@ import Link from '@docusaurus/Link';
 
 # UID2 Tokens and Refresh Tokens
 
-When a publisher sends a user's <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) to the UID2 Operator, whether via one of the UID2 SDKs or the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint, the UID2 Operator returns a <a href="glossary-uid#gl-uid2-token">UID2 token</a> with associated values such as refresh token. The token is an opaque alphanumeric string. Every time a token is generated from DII input, the token value is different. Token values are never the same. Because of the way UID2 works, different instances of activity, on browsers, CTV, and electronic devices such as phone and tablets, can be matched without compromising the privacy of the individual. The token is designed so that it cannot be reverse engineered to arrive at the original email address or phone number.
+When a publisher sends a user's <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) to the UID2 Operator, whether via one of the UID2 SDKs or the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint, the UID2 Operator returns a <a href="glossary-uid#gl-uid2-token">UID2 token</a> with associated values, including a refresh token. The token is an alphanumeric string. Each time a token is generated from DII input, the token value is different. Because of the way UID2 works, different instances of activity, on browsers, CTV, and electronic devices such as phone and tablets, can still be matched to the same <a href="glossary-uid#gl-raw-uid2">raw UID2</a> generated from the user's DII, even though the token might be different. The token is designed to protect against reverse engineering that might reveal the original email address or phone number.
```

Comment:
i feel it is better to just copy the Glossary definition of UID2 token and put it here:

_UID2 tokens are generated from hashed or unhashed email addresses or phone numbers that are converted to raw UID2s and then encrypted._ 


---------------------------------------------------------------------------


_The token is designed to protect against reverse engineering that might reveal the original email address or phone number._

This makes it sound like people who can legit decrypt the token can know the original email/phone value (which isn't true as token can only be decrypted into a raw UID2. But yes if malicious party which has a mapping of raw UID2->original email/phone they can know the original email/phone.

We could reword
_The token is designed to protect the raw UID2 value inside the token so that only participant which have permission to decrypt can reveal the raw UID2 inside it._ 

Or just copy the glossary definition instead of reinventing the wheel here.
https://unifiedid.com/docs/ref-info/glossary-uid#gl-uid2-token

I feel current wording could be confusing and we should change it one way or another - i feel copy from glossary is easier.

Reply:
Majorly updated. from the glossary, to this:

When a publisher sends a user's DII—hashed or unhashed email addresses or phone numbers—to the UID2 Operator, whether via one of the UID2 SDKs or the POST /token/generate] endpoint, the UID2 Operator converts the DII to a raw UID2, encrypts it into a UID2 token, and returns the UID2 token with associated values, including a refresh token. The publisher can then use the UID2 token in the bidstream.